docs(clustered): mention use of `DOCKER_CONFIG` with `kubit local apply` (#5357)
influxdata/docs-v2: content/influxdb/clustered/install/deploy.md

    and related tools on your local machine.

2.  Use the `kubit local apply` command to apply your custom-configured

>     `myinfluxdb.yml` and deploy your InfluxDB Cluster. You should also specify the
>     directory containing your Clustered pull secret credentials, provided by InfluxData,
>     in the `DOCKER_CONFIG` environment variable.

Reviewer: ```suggestion
    `myinfluxdb.yml` and deploy your InfluxDB Cluster.
    Set the `DOCKER_CONFIG` environment variable to the directory path of
    your InfluxDB Clustered pull secret credentials provided by InfluxData.
```
